src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -93,7 +93,7 @@ const ChartsHolder = React.memo(({ region, chartId }: ChartsHolderProps) => {
 
   const [currentExploreMetric, setCurrentExploreMetric] = useState<
     ExploreMetric
-  >(ExploreMetric.WEEKLY_CASES);
+  >(ExploreMetric.HOSPITALIZATIONS);
```

Comment:
I'm not sure why this was set to WEEKLY_CASES before.  It ends up getting overridden (in most cases at least) by this code that sets it to HOSPITALIZATIONS: https://github.com/covid-projections/covid-projections/blob/3ee5232c0f491e18cf86083f92485521a329a150/src/components/Explore/Explore.tsx#L256

So this results in no behavior change.